assets/capability/capabilities/measure_noise.json

```diff
@@ -8,22 +8,28 @@
     "it": "Rumore",
     "sv": "Buller",
     "no": "Støy",
-    "es": "Ruido"
+    "es": "Ruido",
+    "ru": "Шум",
+    "pl": "Hałas"
   },
   "units": {
-    "en": "dB"
+    "en": "dB",
+    "ru": "дБ",
+    "pl": "dB"
   },
   "insights": true,
   "desc": {
-    "en": "Noise in decibel (db)",
-    "nl": "Geluid in decibel (db)",
+    "en": "Noise in decibel (dB)",
+    "nl": "Geluid in decibel (dB)",
     "de": "Lärm in Dezibel (dB)",
     "fr": "Bruit en decibel (dB)",
-    "it": "Rumore in decibel (db)",
-    "sv": "Buller i decibel (db)",
-    "no": "Støy i decibel (db)",
+    "it": "Rumore in decibel (dB)",
+    "sv": "Buller i decibel (dB)",
```

Comment:
Other languages have been changed here, is this correct?

Reply:
I changed the lowercase b to capital B because the unit of sound is dB, not db